demisto_sdk/commands/common/constants.py

```diff
@@ -905,7 +907,12 @@ class GithubContentConfig:
     CONTENT_GITHUB_LINK: str
     CONTENT_GITHUB_MASTER_LINK: str
 
+    IS_GITLAB = False
+    GITLAB_ID: Optional[int] = None
```

Comment:
If you have a GITLAB_ID then is `IS_GITLAB` needed?

Reply:
I'm guess I can check if `GITLAB_ID` is `None`, you think it's better?